packages/block-library/src/separator/edit.js

```diff
@@ -51,6 +52,7 @@ export default function SeparatorEdit( { attributes, setAttributes } ) {
 				{ ...useBlockProps( {
 					className,
 					style: hasCustomColor ? styles : undefined,
+					...( Platform.isNative && { color: styles.color } ),
```

Comment:
My understanding is that we generally discourage inline `Platform` conditionals to avoid increased complexity and web bundle size. Is it possible to avoid this mobile-specific conditional by relying upon the passed `style` prop and referencing `style.color` or `style.backgroundColor` within the `HorizontalRule` implementation? I.e. attempting to align with the web API.